src/sgame/sg_bot_public.h

```diff
@@ -69,6 +70,7 @@ void G_BotCleanup();
 void G_BotFill( bool immediately );
 void G_BotRemoveObstacle( qhandle_t handle );
 void G_BotUpdateObstacles();
+std::string btos( gentity_t *bot );
```

Comment:
I huh, am not sure having a global function with a undescriptive name is a good idea. Even if `vtos` or `etos` do exist

Reply:
Renamed.